optimum/intel/openvino/modeling_base_seq2seq.py

```diff
@@ -309,5 +319,41 @@ def _from_transformers(
 
         return cls._from_pretrained(save_dir, **kwargs)
 
+    def _reshape(self, model: openvino.runtime.Model, batch_size: int, sequence_length: int, is_decoder=True):
+        shapes = {}
+        for inputs in model.inputs:
+            shapes[inputs] = inputs.get_partial_shape()
+            shapes[inputs][0] = batch_size if not is_decoder else -1
+            if inputs.get_any_name().startswith("past_key_values"):
+                shapes[inputs][2] = -1
+            elif is_decoder and not inputs.get_any_name().startswith("encoder"):
+                shapes[inputs][1] = -1
```

Comment:
I think we should print a warning message here: "WARNING: Decoder part of the model does not support static shapes and kept dynamic."

Reply:
I have added it !